probe-rs/src/core/mod.rs

```diff
@@ -132,6 +132,48 @@ pub enum RegisterValue {
 }
 
 impl RegisterValue {
+    /// A helper function to add a fixed number of bytes to a register value.
+    pub fn add_bytes(&mut self, bytes: usize) -> Result<(), Error> {
```

Comment:
I don't think add bytes is a good name here. We don't really add any bytes. We merely increase an address.

Reply:
Valid comment. How's this?